src/relay/backend/contrib/cmsisnn/scalar_to_tensor_constant.cc

```diff
@@ -67,8 +67,7 @@ class ScalarToTensorConstantMutator : public MixedModeMutator {
     Expr final_call = post;
     call = post.as<CallNode>();
 
-    // Create a new variable argument that is of the same shape as the neighbouring argument
-    // in the binary op. This needs to be done only when one of the arguments is a scalar.
+    // Substitute scalar variable with a tensor variable in qnn.add and qnn.mul ops
```

Comment:
nit : I feel here we could just say Substitute scalar variable with a tensor variable where appropiate, because it seems what is being replaced is an implementation detail of ReplaceScalarWithTensorVariable

Reply:
Makes sense.